index.js

```diff
@@ -123,6 +123,40 @@ function fastifyMultipart (fastify, options, done) {
         req.body = body
       }
     })
+
+    // The following is not available on old Node.js versions
+    // so we must skip it in the test coverage
+    /* istanbul ignore next */
+    if (globalThis.FormData) {
+      fastify.decorateRequest('formData', async function () {
+        const formData = new FormData()
+        for (const key in this.body) {
+          const value = this.body[key]
+          if (Array.isArray(value)) {
+            for (const item of value) {
+              await append(key, item)
+            }
+          } else {
+            await append(key, value)
+          }
+        }
+
+        async function append (key, entry) {
```

Comment:
there are no awaits, does this need to be async?

Reply:
Yeah I thought the same for a second: https://github.com/fastify/fastify-multipart/pull/522/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R149

Reply:
thanks :D